src/edu/gatech/cs2340/trydent/gui/TrydentButton.java

```diff
@@ -0,0 +1,63 @@
+package edu.gatech.cs2340.trydent.gui;
+
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.scene.control.Button;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+/**
+ * A basic button
+ * @author sparky
+ *
+ */
+public class TrydentButton extends TrydentGUIWrapper<Button> {
```

Comment:
This is already in the trydent package, so maybe we should just call it Button.

Or maybe we want to avoid name confusion. Opinions on naming?